templates/playlist/playlist-client.php

```diff
@@ -41,56 +41,61 @@
     
 	<div id="mdjm-playlist-form">
     	<?php do_action( 'mdjm_playlist_form_top', $mdjm_event->ID ); ?>
+
         
-        <?php if( $mdjm_event->playlist_is_open() ) : ?>
-        
-            <form id="mdjm-playlist-form" name="mdjm-playlist-form" action="" method="post">
-                <?php wp_nonce_field( 'add_playlist_entry', 'mdjm_nonce', true, true ); ?>
-                <?php mdjm_action_field( 'add_playlist_entry' ); ?>
-                <input type="hidden" id="entry_event" name="entry_event" value="<?php echo $mdjm_event->ID; ?>" />
-                
-                <table id="mdjm-playlist-form-table">
-                    <tr>
-                        <td class="mdjm-playlist-song-cell">
-                            <label for="entry_song"><?php _e( 'Song', 'mobile-dj-manager' ); ?></label><br />
-                            <input type="text" name="entry_song" id="entry_song" data-placeholder="<?php _e( 'Song Name', 'mobile-dj-manager' ); ?>" required />
-                        </td>
-                        
-                        <td class="mdjm-playlist-artist-cell">
-                            <label for="entry_artist"><?php _e( 'Artist', 'mobile-dj-manager' ); ?></label><br />
-                            <input type="text" name="entry_artist" id="entry_artist" data-placeholder="<?php _e( 'Artist Name', 'mobile-dj-manager' ); ?>" required />
-                        </td>
-                        
-                        <td class="mdjm-playlist-category-cell">
-                            <label for="entry_category"><?php _e( 'Category', 'mobile-dj-manager' ); ?></label><br />
-                            <?php mdjm_playlist_category_dropdown(); ?>
-                        </td>
-                        
-                        <td class="mdjm-playlist-djnotes-cell">
-                            <label for="mdjm_playlist_djnotes"><?php printf( __( 'Notes for your %s', 'mobile-dj-manager' ), '{artist_label}' ); ?></label><br />
-                            <textarea name="entry_djnotes" id="entry_djnotes" data-placeholder="<?php printf( __( 'Notes for your %s', 'mobile-dj-manager' ), '{artist_label}' ); ?>"></textarea>
-                        </td>
-                    </tr>
-                    <tr>
-                    	<td class="mdjm-playlist-addnew-cell" colspan="4">
-                            <input type="submit" name="entry_addnew" id="entry_addnew" value="<?php _e( 'Add to Playlist', 'mobile-dj-manager' ); ?>" />
-                        </td>
-                    </tr>
-                </table>
-            </form>
+       	<?php if( $mdjm_event->playlist_is_open() ) : ?>
```

Comment:
Please ensure that each new `if` is indented correctly and that the related `else` and `endif` statements are indented the same so it is easy to read when there is a lot of code.

Reply:
Bah I *knew* my indenting was gonna get me into trouble.  I'm editing code via SSH on a live server. In NANO.  Ha!